wallet/wallet-controller/src/synced_controller.rs

```diff
@@ -418,6 +424,46 @@ impl<'a, T: NodeInterface, W: WalletEvents> SyncedController<'a, T, W> {
         .await
     }
 
+    pub async fn request_send_to_address(
```

Comment:
Please document what this function and the one above it do... very vague. Keep in mind that these functions with these random names will not be recognized easily in the future except by you.